pipes.Rmd

```diff
@@ -37,26 +35,51 @@ If the arguments to a function don't all fit on one line, put each argument on
 its own line and indent:
 
 ```{r, eval = FALSE}
-iris %>%
-  group_by(Species) %>%
+# Good
+iris |>
   summarise(
     Sepal.Length = mean(Sepal.Length),
-    Sepal.Width = mean(Sepal.Width)
+    Sepal.Width = mean(Sepal.Width),
+    .by = Species
   )
+
+# Bad
+iris |>
+  summarise(Sepal.Length = mean(Sepal.Length), Sepal.Width = mean(Sepal.Width), .by = Species)
+```
+
+For data analysis, we recommend using the pipe whenever a function needs to span multiple lines, even if it's only a single step.
```

Comment:
nit: I would consider this as a follow-up PR, since it's not strictly about the topic at hand (change `%>%` to `|>`). Keeps the git history cleaner.

I think at a minimum the PR title should be more representative of what's actually included.